.gitignore

```diff
@@ -96,3 +96,4 @@ src/TestResults
 test/core/customast/test.txt
 test/core/files/test.png
 test/core/migration/writetext.txt
+src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs
```

Comment:
Thanks @mjkkirschner , looked around and it was already part of the `.gitignore`, see a [few lines above ](https://github.com/radumg/Dynamo/blob/930873512557ed43087a2d1868fab198d6d85ae5/.gitignore#L74)- my GitHub added it when i ignored it. I untracked the file in my repo now, just need to do the same for this file too.